packages/libs/e2e-test-utils/src/lib/index.ts

```diff
@@ -18,3 +18,4 @@ export { ensureUnexpiredCapacityToken } from './ensure-capacity-credit';
 export { getEnv } from './env';
 export { getChainHelpers } from './chain';
 export { createRandomVincentWallets } from './create-random-vincent-wallets';
+export { setupVincentDevelopmentEnvironment } from './setup-vincent-development-environment';
```

Comment:
Need to export `SetupResult`, and we should probably rename to `VincentDevEnvironment` or similar